appveyor.yml

```diff
@@ -0,0 +1,28 @@
+version: "{build}"
+
+shallow_clone: true
+
+environment:
+  matrix:
+    - NODEJS_VERSION: "8"
+      PLATFORM: x86
```

Comment:
I think we're fine without x86 specific testing, let's just drop this.  It can always be added later if we find that it's needed.